test/jit/test_symbolic_shape_analysis.py

```diff
@@ -573,3 +573,14 @@ def too_many_inputs(x: List[int], y: List[int], z: Any, z2: Any):
             torch._C._jit_register_shape_compute_graph_for_node(node, too_many_inputs.graph)
 
         self.assertTrue("fewer arguments than schema" in str(error.exception))
+
+    def test_cross_entropy_loss(self):
+        input = torch.randn((8, 2))
+        target = torch.randint(high=2, size=(8,))
+
+        def foo(input, target):
+            return torch.ops.aten.cross_entropy_loss(input, target)
+
+        t = torch.jit.script(foo, [input, target])
+        torch._C._jit_pass_propagate_shapes_on_graph(t.graph)
+        self.assertEqual(next(t.graph.outputs()).type().symbolic_sizes(), [])
```

Comment:
same comment as in #98144 - can we test the actual sizes?

also based on discussions with @eellison - in the future do you think would be possible to leave these new shape functions in the torch-mlir repo? we're not aware of other users who use them - and that way you don't have to wait for our reviews, and you can choose your own strategy for testing testing the registered shape functions. cc @silvasean

Reply:
Would you say that the symbolic shape analysis is deprecated then @eellison @davidberard98 ?

Reply:
Hmm, I don't know about deprecated, but maybe stable is a better term ? We really appreciate the previous contributions. At this point we are focusing our efforts on PT 2.0. 

I will follow up on ways torch-mlir can replicate some of the testing that we have in core for shape functions. With OpInfos and TorchDispatchMode we should be able to get pretty thorough coverage easily.